pkg/generate/app/sourcelookup.go

```diff
@@ -74,17 +74,21 @@ func IsPossibleSourceRepository(s string) bool {
 // IsRemoteRepository checks whether the provided string is a remote repository or not
 func IsRemoteRepository(s string) bool {
 	if !s2igit.New(s2iutil.NewFileSystem()).ValidCloneSpecRemoteOnly(s) {
+		glog.V(5).Infof("%s is not a valid remote git clone spec", s)
```

Comment:
@gabemontero see the issue i assigned you a few days ago for more context on this behavior and the challenges it presents in terms of how to present errors to users.